bucket/desmume.json

```diff
@@ -0,0 +1,37 @@
+{
+    "version": "0.9.13",
+    "description": "Nintendo DS emulator",
+    "homepage": "https://github.com/TASEmulators/desmume",
+    "license": "GPL-2.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/TASEmulators/desmume/releases/download/release_0_9_13/desmume-0.9.13-win64.zip",
+            "hash": "cbf710b8e6b29f0904e289a9b9a895548a2d7c7af5a3145c8279b97a69a09276",
+            "shortcuts": [
+                [
+                    "DeSmuME_0.9.13_x64.exe",
+                    "DeSmuME"
+                ]
+            ]
+        }
+    },
+    "pre_install": "if (!(Test-Path \"$persist_dir\\desmume.ini\")) { New-Item \"$dir\\desmume.ini\" | Out-Null }",
+    "persist": "desmume.ini",
+    "checkver": {
+        "url": "https://api.github.com/repos/TASEmulators/desmume/releases",
+        "regex": "/desmume-([\\d.]+)-win64\\.zip"
+    },
```

Comment:
Can't the standard `github` checkver be used here?

Reply:
Doesn't look like it. Release tags aren't normal format. `release_0_9_13`